File: client-java/src/main/java/com/lei/model/Attribute.java
package com.lei.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author lizhi
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Attribute {
    /**
     * 私有属性id格式 attribute:resourceId:uuid
     */
    private String id;

    private String type;

    /**
     * 针对那一个资源
     */
    private String resourceId;

    /**
     * 属性发布者
     */
    private String ownerId;

    private double money;

    private String key;
    private String value;
    /**
     * 生效时间
     */
    private String notBefore;
    /**
     * 失效时间
     */
    private String notAfter;


}

File: chaincode-go/model/attribute.go
package model

type Attribute struct {
	Id         string  `json:"id"`
	Type       string  `json:"type"`
	ResourceId string  `json:"resourceId"`
	Owner      string  `json:"ownerId"`
	Key        string  `json:"key"`
	Value      string  `json:"value"`
	NotBefore  string  `json:"notBefore"`
	NotAfter   string  `json:"notAfter"`
	Money      float64 `json:"money"`
}
